src/commands/profiles/create.js

```diff
@@ -30,9 +23,6 @@ class ProfilesCreate extends BaseCommand {
   async run() {
     await super.run();
 
-    // Eagerly load up the credential store. No need to proceed if this fails.
-    await this.secureStorage.loadKeytar();
```

Comment:
Any idea why we used to do this previously? Is it not effect the case where profiles are in both keytar and config file?

Reply:
Previously since all keys were in keytar, it made sense to load this early on and then read the credentials, but now, since we are storing in file by default, we shouldn't do this eager loading. They will be loaded from keytar only when necessary.

Reply:
Okay.